test/e2e/framework/service_util.go

```diff
@@ -236,7 +240,8 @@ func (j *ServiceTestJig) CreateServiceWithServicePort(labels map[string]string,
 	return j.Client.CoreV1().Services(namespace).Create(service)
 }
 
-func (j *ServiceTestJig) ChangeServiceType(namespace, name string, newType v1.ServiceType, timeout time.Duration) {
+// UpdateServiceType updates the given service's ServiceType to the given newType.
+func (j *ServiceTestJig) UpdateServiceType(namespace, name string, newType v1.ServiceType, timeout time.Duration) {
```

Comment:
Why do we need to rename the method? Is that for consistency in `test/e2e/framework` ?
This question comes because we need to get wider consensus under `test/e2e` by comparing changing `test/e2e/framework` only.

Reply:
I’m not sure that policy of rename has consistency in `test/e2e/framework`.
I’ll return it to original because that motivation is to fix golint failures.

Reply:
done.